src/create-reducer.spec.snap.ts

```diff
@@ -7,10 +7,12 @@ import { getType } from './get-type';
 const add = createAction('ADD')<number>();
 const increment = createAction('INCREMENT')();
 const decrement = createAction('DECREMENT')();
+const reduxInit = createAction('@@redux/INIT.1')();
```

Comment:
reduxInit cannot be handled as it is implementation details of Redux not a public API